generate_workflow.py

```diff
@@ -169,9 +170,11 @@ def get_steps(build, build_set):
 
 def patch_series(tree_name):
     ci_folder = pathlib.Path(__file__).resolve().parent
-    mbox = ci_folder.joinpath("patches", "%s.mbox" % tree_name)
-    mbox_rel_path = mbox.relative_to(ci_folder).as_posix()
-    return " --patch-series {}".format(mbox_rel_path) if mbox.exists() else ""
+    patches_folder = pathlib.Path("patches").joinpath(tree_name)
+    patch_files = glob.glob(
+        ci_folder.joinpath(patches_folder).as_posix() + "/*.patch")
+    return " --patch-series {}".format(
+        patches_folder.as_posix()) if patch_files else ""
```

Comment:
there's duplication between the additions to generate_workflow.py and generate_tuxsuite.py. Can you move this to utils.py and try to reuse more code between the two?